engine/admin/index.md

```diff
@@ -84,6 +84,15 @@ non-responsive, you can also
 threads to be added to the daemon log by sending the `SIGUSR` signal to the
 Docker daemon.
 
+### Out Of Memory Exceptions (OOME)
+
+If your containers attempt to use more memmory than the system has available,
+you may experience an Out Of Memory Exception (OOME) and a container, or the
+Docker daemon, might be killed by the kernel OOM killer. To prevent this from
+happening. ensure that your application runs on hosts with adequate memory and
+see
+[Understand the risks of running out of memory](/engine/admin/resource_constraints.md#understand-the-risks-of-running-out-of-memory).
```

Comment:
We should also think about pointing people to the info on resource constraints in the Compose file reference.